optimum/onnxruntime/base.py

```diff
@@ -527,7 +534,7 @@ def compute_past_key_values_output_shapes(
     ) -> Dict[str, int]:
         batch_size = input_ids.size(0)
         num_attention_heads = self.normalized_config.num_attention_heads
-        embed_size_per_head = self.normalized_config.hidden_size // num_attention_heads
+        embed_size_per_head = self.normalized_config.hidden_size // self.normalized_config.num_attention_heads
```

Comment:
```suggestion
        embed_size_per_head = self.normalized_config.hidden_size // num_attention_heads
```

Reply:
https://github.com/huggingface/transformers/blob/v4.34.0/src/transformers/models/mistral/modeling_mistral.py#L223-L225

Reply:
where :
```python
self.head_dim = config.hidden_size // config.num_attention_heads
self.num_key_value_heads = config.num_key_value_heads
```

Reply:
@echarlaix Here would it make sense to leave it as `embed_size_per_head = self.normalized_config.hidden_size // num_attention_heads` given that `num_attention_heads = self.normalized_config.num_attention_heads` is defined in the line just above?

Reply:
Good catch, added it in https://github.com/huggingface/optimum/pull/1425/commits/3d0ac048d92199b23f736f2794ad970b40f81a36